web/templates/partials/tiles/law/wet_op_de_huurtoeslag/TOESLAGEN.html

```diff
@@ -1,25 +1,27 @@
 {% extends "partials/tiles/base_tile.html" %}
 {% block tile_content %}
-    {% if result.is_eligible %}
-        {% include "partials/tiles/law/wet_op_de_huurtoeslag/TOESLAGEN/computation.html" %}
-    {% else %}
-        <div class="text-sm leading-6 text-gray-700 mt-4">
-            <p class="mb-3">
-                U voldoet niet aan alle voorwaarden voor huurtoeslag.
-                Dit kan verschillende redenen hebben, zoals:
-            </p>
-            <ul class="list-disc pl-5 space-y-2">
-                <li>Uw inkomen is te hoog (meer dan €44.000 voor alleenstaanden)</li>
-                <li>Uw vermogen is te hoog (meer dan €36.952 voor alleenstaanden)</li>
-                <li>Uw huur is te hoog (meer dan €885,71)</li>
-                <li>U bent jonger dan 18 jaar</li>
-            </ul>
-            <p class="mt-3">
-                Bekijk voor meer informatie de uitleg hiernaast.
-                U kunt ook kijken op <a href="https://www.belastingdienst.nl/huurtoeslag"
+    {% if not missing_required %}
+        {% if result.is_eligible %}
+            {% include "partials/tiles/law/wet_op_de_huurtoeslag/TOESLAGEN/computation.html" %}
+        {% else %}
+            <div class="text-sm leading-6 text-gray-700 mt-4">
+                <p class="mb-3">
+                    U voldoet niet aan alle voorwaarden voor huurtoeslag.
+                    Dit kan verschillende redenen hebben, zoals:
+                </p>
+                <ul class="list-disc pl-5 space-y-2">
+                    <li>Uw inkomen is te hoog (meer dan €44.000 voor alleenstaanden)</li>
```

Comment:
I don't think we should hardcode these values. If we'd like to mention reasons with specific values, we should take them from the law.

Reply:
Eens, maar dit zat al in de code, heb hier alleen de logic gewijzigd (toch?)

Reply:
Oh, I completely missed that. Sorry, I agree!